test/sanity_test.go

```diff
@@ -0,0 +1,141 @@
+package test
+
+import (
+	"flag"
+	"fmt"
+	"github.com/anticycle/anticycle/pkg/anticycle"
```

Comment:
Maybe it would be a good idea to group imported packages into three blocks:

1. Standard library - for example `fmt` 
2. Internal packages - for example `github.com/anticycle/anticycle/pkg/anticycle`
3. External packages - for example `github.com/stretchr/testify/assert`

What do you think about it?

Reply:
Yes but this kind of sorting require linter. If you know any lightweight linter and you want to configure it, be my guest. Create new issue and attach yourself to it.